presto-main/src/main/java/io/prestosql/sql/planner/PlanFragmenter.java

```diff
@@ -130,16 +130,17 @@ private void sanityCheckFragmentedPlan(SubPlan subPlan, WarningCollector warning
         subPlan.sanityCheck();
         int fragmentCount = subPlan.getAllFragments().size();
         if (fragmentCount > maxStageCount) {
-            throw new PrestoException(QUERY_HAS_TOO_MANY_STAGES, format(
-                    "Number of stages in the query (%s) exceeds the allowed maximum (%s). " + TOO_MANY_STAGES_MESSAGE,
+            throw new PrestoException(QUERY_HAS_TOO_MANY_STAGES, format("Number of stages in the query (%s) exceeds the allowed maximum (%s). %s",
```

Comment:
Keep the first line of the string wrapped, so that it's consistent with the one below.